src/NUnitEngine/nunit.engine/Internal/ResultHelper.cs

```diff
@@ -112,8 +118,9 @@ public static void InsertEnvironmentElement(this XmlNode resultNode)
             env.AddAttribute("machine-name", Environment.MachineName);
             env.AddAttribute("user", Environment.UserName);
             env.AddAttribute("user-domain", Environment.UserDomainName);
-            env.AddAttribute("culture", System.Globalization.CultureInfo.CurrentCulture.ToString());
-            env.AddAttribute("uiculture", System.Globalization.CultureInfo.CurrentUICulture.ToString());
+            env.AddAttribute("culture", CultureInfo.CurrentCulture.ToString());
+            env.AddAttribute("uiculture", CultureInfo.CurrentUICulture.ToString());
+            env.AddAttribute("os-architecture", Environment.GetEnvironmentVariable("PROCESSOR_ARCHITECTURE"));
```

Comment:
That's a good catch. It's not set on Ubuntu and neither is Ubuntu.

@agray Since you're working this one, what do you think we should do? Include it only if set? Find some other way to get the processor architecture? If the latter, we may want to add a property to the NUnit.Env class.

Or maybe we don't want to use the environment at all but figure out an alternative way to get processor architecture across platforms. As a reminder, we're looking for something like x86, x64, etc. here.

So much for easy fixes. :-)